packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -153,48 +206,18 @@ const ModalContent = forwardRef((props, ref) => {
           the aria-label value when the modal is opened
         */
       }
-      <div
-        {...customProps}
-        tabIndex={platformIsiOS || isCalledFromNotificationDialog ? '-1' : '0'}
-        aria-label={ariaLabel}
-        aria-labelledby={ariaLabelledBy}
-        aria-describedby={ariaDescribedBy}
-        className={modalClassName}
-        role={role}
-        ref={ref}
-      >
-        <div className={modalContainerClassName} ref={setModalFocusElementRef} data-terra-abstract-modal-begin tabIndex="-1">
-          {(!isCalledFromNotificationDialog) && (
-            <FormattedMessage id="Terra.AbstractModal.BeginModalDialog">
-              {text => {
-                // In the latest version of react-intl this param is an array, when previous versions it was a string.
-                let useText = text;
-                if (Array.isArray(text)) {
-                  useText = text.join('');
-                }
-                return (
-                  <VisuallyHiddenText text={useText} />
-                );
-              }}
-            </FormattedMessage>
-          )}
-          {children}
-          {(!isCalledFromNotificationDialog) && (
-            <FormattedMessage id="Terra.AbstractModal.EndModalDialog">
-              {text => {
-                // In the latest version of react-intl this param is an array, when previous versions it was a string.
-                let useText = text;
-                if (Array.isArray(text)) {
-                  useText = text.join('');
-                }
-                return (
-                  <VisuallyHiddenText text={useText} />
-                );
-              }}
-            </FormattedMessage>
-          )}
+      {enableFocusLock
+        && (
+        <FocusLock>
+          {modalContent}
+        </FocusLock>
+        )}
+      {!enableFocusLock
+        && (
+        <div>
+          {modalContent}
         </div>
-      </div>
+        )}
```

Comment:
Seems like a ternary operator would be better in this scenario.  Or the modalContent variable could be updated if enableFocus lock is true and then rendered.

Reply:
Done.